tests/CMakeLists.txt

```diff
@@ -25,6 +26,7 @@ SET(TESTS
   eiquadprog-both
   )
 
+ADD_DEFINITIONS(-Wno-c99-extensions)
```

Comment:
I don't see any warning with g++ 10.1.
With clang 10.0, I have some warnings about unused-parameters & sign-conversion.
Are those ones which are bothering you ? I think both of them should be fixed in the code, not silenced. They are really easy to fix.